### PR TITLE
JS Minify Visual Improvements

### PR DESCRIPTION
**Pull request by moollaza**
##### What does your Pull Request do (check all that apply)?

- [x] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [x] Enhancement: **`{IA Name}: {Description of Improvements}`**

###### Description of changes

- Better button sizing and position for desktop and mobile
- Adjust  textarea height for mobile
- Add DDG Form style classes to textarea
- Add placeholder text to input
- Adjust subtitle text

UI now looks like:

js_minify_at_duckduckgo

js_minify_at_duckduckgo

###### People to notify (@mention interested parties)
@sahildua2305 @MariagraziaAlastra 

---

Instant Answer Page: https://duck.co/ia/view/js_minify

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/js_minify/js_minify.css [hunk 1]
```diff
@@ -1,18 +1,32 @@
...
+.zci--js_minify .js_minify--wrap {
+    margin-bottom: 1.5em;
 }
 
-.zci--js_minify #js_minify__input,
-.zci--js_minify #js_minify__output {
-    width: 45%;
+.zci--js_minify textarea {
     font-family: 'monospace';
+    overflow: auto; # Hide default scrollbars on IE
```
Comment (MariagraziaAlastra) [inline share/goodie/js_minify/js_minify.css]: @moollaza use /* ... */ instead?

File: share/goodie/js_minify/js_minify.js [hunk 2]
```diff
@@ -45,9 +47,12 @@ DDH.js_minify.build = function(ops) {
...
                         // 'pointer'
-                        $minifyButton.text('Minify');
-                        $minifyButton.prop('disabled', false);
-                        $minifyButton.css('cursor', 'pointer');
+                        $minifyButton
+                            .text('Minify Code')
+                            .prop('disabled', false)
+                            .css('cursor', 'pointer')
+                            .removeClass('btn--skeleton')
+                            .addClass('btn--primary')
```
Comment (MariagraziaAlastra) [inline share/goodie/js_minify/js_minify.js]: Don't you need a semicolon here?

**Comment by MariagraziaAlastra** [comment]
@moollaza Left you a couple really minor comments, looks good to me and it's on Beta. 
If it's ready we could merge and then I could release Goodies again...

**Comment by sahildua2305** [comment]
@moollaza This is awesome! :blue_heart: 

A lot to learn from this. Thanks a lot.

**Comment by sahildua2305** [comment]
I will include the CSS minify action in this IA once this PR is merged.

/cc @moollaza @MariagraziaAlastra

File: share/goodie/js_minify/js_minify.css [hunk 3]
```diff
@@ -1,18 +1,32 @@
...
+
+.zci--js_minify .btn {
+    cursor: default;
+}
+
+.zci--js_minify .js_minify--action_box button {
+    padding-left: 0;
+    padding-right: 0;
+    width: 100%;
 }
```
Comment (sahildua2305) [inline share/goodie/js_minify/js_minify.css]: @moollaza This styling block should be only for mobile, right?

**Comment by sahildua2305** [comment]
@moollaza We can merge this after minor changes as suggested in comments. Would you please update the PR?

**Comment by moollaza** [comment]
@MariagraziaAlastra @sahildua2305 thanks for the feedback! I've updated accordingly

**Comment by vikashvverma** [comment]
@moolaza Seems really good. Do we have  `js beautifier` and `json lint` as well?

**Comment by moollaza** [comment]
@sahildua2305 would you like to add a `Clear` button to this IA as well?

> @moolaza Seems really good. Do we have js beautifier and json lint as well?

@vikashvverma I believe @sahildua2305 is working on the beautifier? He also completed a JSON Linter very recently 👍

**Comment by vikashvverma** [comment]
@sahildua2305 Validate button should remain disabled when there is no input

**Comment by vikashvverma** [comment]
@moollaza I just saw, Would be nice to have `json lint` and `jsonlint` as triggers

**Pull request merged**